### PR TITLE
JOUR-642 Search for applink urls and add to link data

### PR DESCRIPTION
**Pull request by yntema**
Look for ios and android applink urls:
`<meta property="al:ios:url" content="applinks://docs" />`
`<meta property="al:android:url" content="applinks://docs" />`

and includes them in link data if found:
`{
    $ios_deeplink_path: 'applinks://docs',
    $android_deeplink_path: 'applinks://docs'
}`

JIRA Ticket: https://branch.atlassian.net/browse/JOUR-642

**Reviewers: ** @jsaleigh @TheMarshal99

### CODE REVIEW

**Comment by zeeshabnc** [comment]
Can you reference the JIRA in the commit please: JOUR-642

**Comment by zeeshabnc** [comment]
Typo in the commit message: { $ios_deeplink_path: 'applinks://docs', $ios_deeplink_path: 'applinks://docs' }
One of them should be $android_deeplink_path

**Comment by jsaleigh** [comment]
@yntema a few of the tests failed, for instance: https://saucelabs.com/jobs/a8be8c08da65439d9b490920bde7b559

This does not seem to be the normal `couldn't connect to SauceLabs` issue

File: src/1_utils.js [hunk 1]
```diff
@@ -441,26 +441,37 @@ utils.scrapeOpenGraphContent = function(property, content) {
...
-			params[split[2]] = metas[i].getAttribute('content');
+			if ((split.length === 3) && (split[0] === 'branch') && (split[1] === 'deeplink')) {
+				params[split[2]] = metas[i].getAttribute('content');
+			}
+		}
+		else if (metas[i].getAttribute('property')) {
+			var property = metas[i].getAttribute('property');
+
+			if (property === 'al:ios:url') {
+				params['$ios_deeplink_path'] = metas[i].getAttribute('content');
```
Comment (TheMarshal99) [inline src/1_utils.js]: Double check, but I'm pretty sure App Links urls include the URI scheme. (e.g. my-scheme://host/pathname)

We'll wanna strip the scheme out before setting it to the $deeplink_path.

**Comment by yntema** [comment]
@jsaleigh I think I know why the tests are failing. I'll get it fixed

File: src/1_utils.js (hunk 1 above)
Comment (yntema) [inline src/1_utils.js]: Thanks! @TheMarshal99

**Comment by jsaleigh** [comment]
👍

**Pull request merged**